.npmignore

```diff
@@ -0,0 +1,11 @@
+node_modules
```

Comment:
We use the `files` property in `package.json`, which produces an *opt-in* list of files we want to ship.  As of today, that's only the contents of `build/src` (along with defaults like `package.json`, `README.md`, etc).  You probably don't need this at all!

Reply:
Great, I didn't see that, will remove `.npmigore` :)

Reply:
Done